lib/runtime/formatter.js

```diff
@@ -42,7 +43,7 @@ function formatEditorTextInRange (editor, range, text) {
         dismissable: true
       })
     } else {
-      if (marker.isValid()) {
+      if (marker.isValid() || force) {
```

Comment:
When will those `force` options be used ?

Reply:
Never, I just forgot to remove this from an earlier iteration.